app/client/components/GridView.js

```diff
@@ -909,21 +909,22 @@ GridView.prototype.makeHeadersFromRow = async function(selection) {
     return;
   }
   const record = this.tableModel.tableData.getRecord(selection.rowIds[0]);
-  const formatter = selection.fields[0].formatter();
   const actions = this.viewSection.columns.peek().reduce((acc, col) => {
```

Comment:
Can we use `viewSection.viewFields` here instead, and use the field's formatter? Fields can have separate settings from the ones saved to their column.

Reply:
I made the change your require, but for the ref column we use the visibleColFormatter as we don't have access to the field's formatter.
In what case the field's formatter is different from the column's one ?